src/Api/RetryTrait.php

```diff
@@ -46,11 +50,23 @@ private function retry(callable $callback, int $maxRetries = 3, bool $addJitter
 
     private function wait(int $attempt, bool $addJitter): void
     {
-        if ($attempt > 0) {
-            $waitTime = 1 == $attempt ? 100 : 2 ** $attempt * 100; // exponential strategy
-            $waitTime = $addJitter ? random_int(0, $waitTime) : $waitTime; // adding randomness
-            // wait
-            usleep($waitTime * 1000);
+        if ($attempt <= 0) {
+            return;
+        }
+
+        // exponential strategy
+        $waitTime = (1 + 2 ** $attempt) * 100;
+
+        if ($addJitter) {
+            // jitter by +/- 25%
+            $waitTime *= random_int(75, 125) / 100;
         }
+
+        usleep((int) $waitTime * static::backupTimeBaseFactor());
+    }
+
+    private function backupTimeBaseFactor(): int
```

Comment:
nice idea

Reply:
i am note sure about the method name, couldn't come up with something more suiteable :thinking:

Reply:
Maybe just `getBaseFactor()`?

Reply:
i have settled for `getWaitTimeFactor()` to make it what it will return :)